src/data_files/headers-order.json

```diff
@@ -0,0 +1,80 @@
+{
+    "safari": [
+        "Connection",
+        "sec-ch-ua",
+        "sec-ch-ua-mobile",
+        "upgrade-insecure-requests",
+        "Upgrade-Insecure-Requests",
+        "user-agent",
+        "User-Agent",
+        "accept",
+        "Accept",
+        "sec-fetch-site",
+        "sec-fetch-mode",
+        "sec-fetch-user",
+        "sec-fetch-dest",
+        "Sec-Fetch-Mode",
+        "Sec-Fetch-Dest",
+        "Sec-Fetch-Site",
+        "Sec-Fetch-User",
+        "accept-encoding",
+        "Accept-Encoding",
+        "accept-language",
+        "Accept-Language",
+        "dnt",
+        "DNT",
+        "te"
+    ],
+    "chrome": [
+        "Connection",
+        "sec-ch-ua",
+        "sec-ch-ua-mobile",
+        "upgrade-insecure-requests",
+        "Upgrade-Insecure-Requests",
+        "user-agent",
+        "User-Agent",
+        "accept",
+        "Accept",
+        "sec-fetch-site",
+        "sec-fetch-mode",
+        "sec-fetch-user",
+        "sec-fetch-dest",
+        "Sec-Fetch-Mode",
+        "Sec-Fetch-Dest",
+        "Sec-Fetch-Site",
+        "Sec-Fetch-User",
+        "accept-encoding",
+        "Accept-Encoding",
+        "accept-language",
+        "Accept-Language",
+        "dnt",
+        "DNT",
+        "te"
+    ],
+    "firefox": [
```

Comment:
In Firefox, when I show Raw headers, they are in this order, which is different from the order shown here.

```
GET /apify/header-generator/pull/6/files/83a049bad3c8ac6c912c6aea2fc6bf1be436d104..5871a061e28123d2eeba8c9737d20f565f687e04 HTTP/2
Host: github.com
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:87.0) Gecko/20100101 Firefox/87.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Referer: https://github.com/notifications?query=
DNT: 1
Connection: keep-alive
Cookie: _octo=GH1.1.366724853.1616097798; logged_in=yes; _device_id=015f0382146acc2e69df76f60f76281e; user_session=8aY-3pUchRPn8wLs0SZZasp8f4_xhsEd9DWUP_IMHpqWq6W0; __Host-user_session_same_site=8aY-3pUchRPn8wLs0SZZasp8f4_xhsEd9DWUP_IMHpqWq6W0; dotcom_user=mnmkng; tz=Europe%2FPrague; _gh_sess=rD8I0rPytTa81yEqXbTDOwjsNtjcepm786c25riESJcLl9g3h7ML%2FHJZuvSFUK%2BXrKZo9RQLZ3sAtMeDINYAbBLCd0QEBhU2EC9GbOCQmLEdnhTKycY8MnM7DAtrOWBIeG6UIe3VPHYv82eAoTYsP9M5G%2F1M3uGP1x%2BzNy1Z0Ah0y18Y1YhoAunN8Q9daSup3TLzKvJLITuk8tzxjYxRxEYsxDWHunq28eU8guIHwjVRWcUAxAQkYaglkkEXmqilcK1pt%2BbFn5vwMGwYEGtBREvtBKdlZM%2FJ9gYNiZlM71wjq2Vjyum9OZ67yrEC1L09M6dUFIBygUrNJq5UDbchx0Z8Xo5dMewOC6z2GsH8pdPENObtOfNI2b8ycgsCQp3X4Tc2Vhx139w%3D--z%2FHVITqaLqJanmEY--jxlOPQIu7%2BHg9bd5lbK8DA%3D%3D; tz=Europe%2FPrague; color_mode=%7B%22color_mode%22%3A%22auto%22%2C%22light_theme%22%3A%7B%22name%22%3A%22light%22%2C%22color_mode%22%3A%22light%22%7D%2C%22dark_theme%22%3A%7B%22name%22%3A%22dark%22%2C%22color_mode%22%3A%22dark%22%7D%7D; has_recent_activity=1
Upgrade-Insecure-Requests: 1
TE: Trailers
```

Also, interestingly, they are Capitalized, even though the connection shows HTTP/2 🤷‍♂️ @petrpatek

Reply:
The orderings are not final yet, that is why I have not re-requested a review. I will finish them later today.

Reply:
The capitalized HTTP 2 is weird, I will check whether we have these cases in the headers we gathered as data.

Reply:
I checked the data and out of the first 38k items, there is not a single HTTP 2 one that has User-Agent filled.